routing_common/vehicle_model.cpp

```diff
@@ -148,17 +148,59 @@ vector<VehicleModel::AdditionalRoadType>::const_iterator VehicleModel::FindRoadT
                  [&type](AdditionalRoadType const & t) { return t.m_type == type; });
 }
 
-string DebugPrint(IVehicleModel::RoadAvailability const l)
+VehicleModelFactory::VehicleModelFactory(
+    CountryParentNameGetterFn const & countryParentNameGetterFn)
+  : m_countryParentNameGetterFn(countryParentNameGetterFn)
+{
+}
+
+shared_ptr<VehicleModelInterface> VehicleModelFactory::GetVehicleModel() const
+{
+  auto const itr = m_models.find("");
+  ASSERT(itr != m_models.end(), (""));
```

Comment:
There is no reason in ```("")```. Please, write a meaningful diagnostic message or get rid of ```""```.